script/configgen.sh

```diff
@@ -16,34 +16,62 @@ if [[ $(uname) == "Darwin" ]]; then
 	SED_EW='[[:>:]]' #End of word in regex
 fi
 
+# ifd-from-ansible is responsible for generating the 'infrastructure-data' that is
+# read by the testnet generator tool.
+ifd-from-ansible() {
+	HOST_PATH=$1
+	OUT_PATH=$2
+	
+	cat <<EOF > $OUT_PATH
+	{
+	  "provider": "digital-ocean",
+	  "network": "0.0.0.0/0",
+	  "instances": {
+EOF
+	
+	lines=`grep '^.*name=.*$' $HOST_PATH | tr " " : `;
+	count=`echo "$lines" | wc -l`
+	
+	i=1
+	for host in  $lines; do
+		ip=`echo $host | cut -d: -f1`
+		name=`echo $host | cut -d: -f2 | sed -n 's/name=\(.*\)/\1/p'`
+	cat <<EOF >> $OUT_PATH
+	    "$name": {
+	      "ip_address": "$ip"
+EOF
+		if [ $i -lt $count ]; then
+	cat <<EOF >> $OUT_PATH
+	    },
+EOF
+		else
+	cat <<EOF >> $OUT_PATH
+	    }
+EOF
+		fi
```

Comment:
Could you align these this line 53 for better readability?
(not the "EOF" lines)